tox.ini

```diff
@@ -10,7 +10,8 @@ deps =
     mock
     pytest
     pytest-cov
-commands =
+    flake8
+commands = flake8
```

Comment:
I tried that, but I'm not sure why, the build doesn't fail when `flake8` fails if I move it to the next line.